### ISSUE TITLE
app icon is a white square

### ISSUE THREAD

**Issue by PaulXiCao** [signalapp/Signal-Android#7957]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The usual app icon disappeared and instead there is just a white square.

### Steps to reproduce
- update to current beta version

### Device info

**Device:** Sony M4 Aqua E2303
**Android version:** 6.0.1
**Signal version:** 4.22.4

### Link to debug log

https://debuglogs.org/e68b227f92c93a0a88a0680d8d0800201cd573509a4852f2624042bbe4e2609f

**Comment by greyson-signal** [comment]
Can you tell me a couple things:

1) Are we talking about the app icon on your home screen?
2) Are you using a custom launcher? If so, which one?
3) Are you experiencing this issue? #7908

**Comment by PaulXiCao** [comment]
concerning the questions ...

1. Yes
2. No
3. Yes, similar. the icon in the accounts tab is also unusual. It's just some kind of blueprint icon.

**Comment by greyson-signal** [comment]
@PaulXiCao I just released the 4.23 beta that might fix this. If you'd like, you can [join the beta](https://support.signal.org/hc/en-us/articles/115000269532-How-do-I-join-Signal-s-beta-) and see if that fixes it, or wait until later this week for 4.23 to go to production users.

**Comment by aapeliv** [comment]
@greyson-signal That fixed it for me.

**Comment by PaulXiCao** [comment]
I just tried out 4.23.
It fixed the blueprint image in the accouts tab (#7908) but the app icon on my home screen is still white. Btw the app icon in my app overview/list is the usual signal icon...

**Comment by greyson-signal** [comment]
@PaulXiCao This is going to sound silly, but have you tried restarting your phone? I remember when I was changing the icon, some parts of the system would cache the old icon until I restarted the phone.

Otherwise, when I changed the icon in #7867, besides adding the adaptive icon, all I did was move the existing icons into `mipmap` directories. So for Android < 8 there really shouldn't be a difference? I wonder if your launcher is doing something odd? I'm really not sure.
